src/Controls/src/Core/Handlers/Items/iOS/SelectableItemsViewController.cs

```diff
@@ -45,23 +45,38 @@ public override void ItemDeselected(UICollectionView collectionView, NSIndexPath
 		// Called by Forms to mark an item selected 
 		internal void SelectItem(object selectedItem)
 		{
-			if (ItemsView?.ItemsSource is null)
-			{
-				return;
-			}
-
+			var originalSource = ItemsView.ItemsSource;
```

Comment:
It's probably excessive but can we set this via pattern matching just to keep the null check in place?